docs/index.md

```diff
@@ -295,20 +295,32 @@ If your aws credential file contains profiles that assume a role via the `source
 aws_access_key_id = AKIA4YFAKEKEYXTDS252
 aws_secret_access_key = SH42YMW5p3EThisIsNotRealzTiEUwXN8BOIOF5J8m
 region = us-west-2
+# this Principal must have permissions to sts:AssumeRole for 'arn:aws:iam::*:role/spc_role'
 
-[role-without-mfa]
-role_arn = arn:aws:iam::123456789012:role/test_assume
+[profile_account_a]
```

Comment:
```suggestion
[role_without_mfa_a]
```

To keep the profile name related to the example, any thoughts on keeping these profile names close to what they was before, but updating it to use underscores instead (to be consistent with other example profile names)? We could also update the `cli-user` profile name to be `cli_user` and the names below in the `#### aws.spc:` section.